ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -542,7 +542,7 @@ default_interfaces:
     mlag_peer_link_allowed_vlans: < vlans as string | default -> "2-4094" >
 
     # IP address pool used for MLAG Peer Link (control link) | *Required when MLAG leafs present in topology.
-    # IP is derived from the node id.
+    # IP is derived from the node id. Assign a subnet that can accomodate at least the total number of l3leafs*2.
```

Comment:
Given that the l3leaf could become anything with custom node keys and that I think if you enable MLAG on some l2leafs you will also need IPs from the mlag_peer_ipv4_pool I would probably change the wording using the total number of MLAG devices / switches time 2

Reply:
Good point, have updated, thanks!